Smaller portion of the REE patch.
We still need to bend the gem installation path until we install 1.9 in /usr/lib/ruby/ just like 1.8.

--- lib/rubygems/defaults.rb.orig	2010-05-16 15:11:19.359858029 +0200
+++ lib/rubygems/defaults.rb	2010-05-16 15:11:43.855868924 +0200
@@ -24,7 +24,7 @@
       File.join(ConfigMap[:rubylibprefix], 'gems',
                 ConfigMap[:ruby_version])
     else
-      File.join(ConfigMap[:libdir], ruby_engine, 'gems',
+      File.join(ConfigMap[:sitedir].gsub(/site_ruby\/?$/, ''), 'gems',
                 ConfigMap[:ruby_version])
     end
   end
